backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/backup/DbEntityCleanupManager.java

```diff
@@ -88,6 +100,7 @@ private void cleanCompletedDbEntities() {
                 new Date(currentTimeMillis - TimeUnit.MILLISECONDS.convert(failedImageTransferTime, TimeUnit.MINUTES));
 
         try {
+            failUnmonitoredBackups();
```

Comment:
couldn't it lead to failing backups when this code is executed after the backup is persisted and before the command is being persisted?
if it's something that can only happen after engine restart, how about moving it to the initialization of the backend?

Reply:
I think both are persisted at the same time, but yes, it's a good point